client/login.go

```diff
@@ -72,7 +72,8 @@ func storeAuthDataFilename() string {
 	return filepath.Join(homeDir, ".snap", "auth.json")
 }
 
-func writeAuthData(userData AuthenticatedUser) error {
+// WriteAuthData saves authentication details for later reuse through ReadAuthData
+func WriteAuthData(userData AuthenticatedUser) error {
```

Comment:
Let's preserve both private for now, please. In `export_test.go`, we can do this:
```
var TestWriteAuth = writeAuthData
var TestReadAuth = readAuthData
```

So these functions will be available for the tests. Their names make it clear when reading code that they're not actually public and not supposed to be used in the implementation.